nixos/modules/programs/captive-browser.nix

```diff
@@ -0,0 +1,116 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.captive-browser;
+
+  package = pkgs.buildGoPackage rec {
+    name = "captive-browser";
+    version = "2018-11-08";
+    goPackagePath = name;
+
+    src = pkgs.fetchFromGitHub {
+      owner  = "FiloSottile";
+      repo   = "captive-browser";
+      rev    = "0f84599ddb36c090cf000681f86321aacb354c4a";
+      sha256 = "0fw4g4fsj9gwm711yj81z64bnmfj2g7wfkhnlsj1nfav0n5kfh40";
+    };
+  };
```

Comment:
Shouldn't we add a package for this in the `pkgs/` tree?

Reply:
I anticipated patches dependend on some ```config.programs.captive-browser.*``` options (to overwrite hardcoded constants, etc)
We can move this to ```pkgs``` if no such patches appear during the testing period.